src/lm.jl

```diff
@@ -90,6 +108,15 @@ function coeftable(mm::LinearModel)
               ["x$i" for i = 1:size(mm.pp.X, 2)], 4)
 end
 
+"""
+    predict(mm, newx)
+
+Returns the predictions from the fitted model for a new set of independent variables.
```

Comment:
``
Return the predictions from a fitted model `mm` based on a matrix of independent variables `newx`. Each row of `newx` represents an observation for which to predict the response, and each column represents an independent variable.
``

Just remove the `Arguments` section.